src/components/users/subscription-details.tsx

```diff
@@ -43,11 +43,11 @@ const SubscriptionDetails: React.FunctionComponent<SubscriptionDetailsProps> =
         {!loading && subscriptionData && (
           <div className="sm:px-6 lg:px-0 lg:col-span-9">
             <section className="mb-32">
-              <div className="p-4 w-full">
-                <div className="border border-accents-1	w-full p rounded-md m-auto my-8 max-w-max-content">
+              <div className="w-full p-4">
+                <div className="w-full m-auto my-8 border rounded-md border-accents-1 p max-w-max-content">
```

Comment:
@DrShpongle do I need to adjust my tailwind extension? Almost every file I touch my extension sorts the classes. Not sure if it's correct

Reply:
or do we need to run it once on the entire code base?

Reply:
I think once and with [tailwind's new prettier plugin](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier).